CHANGELOG.md

```diff
@@ -0,0 +1,107 @@
+# Formerly
+
+## Unreleased
+
+* [DC-1475] Modify Formerly plugin to add reCaptcha validation
```

Comment:
Why is this entire file new?

Reply:
I created it because we did not have the file.

Reply:
strange thats unexpected

Reply:
We used the 'readme.md' file to display update lists